lib/mastodon/media_cli.rb

```diff
@@ -300,4 +307,8 @@ def preload_records_from_mixed_objects(objects)
       end
     end
   end
+
+  def attribute_name(name)
+    name == 'data' ? 'data' : name.singularize
```

Comment:
Could this possibly be fixed by adding `inflect.singular 'data', 'data'` to `config/initializers/inflections.rb`?